src/screens/EmailVerification/index.tsx

```diff
@@ -53,12 +54,30 @@ class EmailVerificationComponent extends React.Component<Props> {
         return (
             <div className="pg-emailverification-container">
                 <div className="pg-emailverification">
-                    <div className="pg-emailverification-title">{title}</div>
+                    {!isMobileDevice && <div className="pg-emailverification-title">{title}</div>}
                     <div className="pg-emailverification-body">
                         <div className="pg-emailverification-body-text">{text}</div>
+                        {
+                            !isMobileDevice &&
+                            <div className="pg-emailverification-body-container">
+                                {emailVerificationLoading ? <Spinner animation="border" variant="primary"/> :
+                                    <button className="pg-emailverification-body-container-button"
+                                            onClick={this.handleClick}>{button}</button>}
+                            </div>
+                        }
+                        {isMobileDevice &&
```

Comment:
```suggestion
                        {isMobileDevice && (
```

Reply:
this is something TSLint should be responsible for